docs/components/board/upboard.md

```diff
@@ -35,7 +36,11 @@ Click **Create component**.
 Edit and fill in the attributes as applicable.
 
 {{% /tab %}}
-{{% tab name="JSON Template" %}}
+{{% tab name="JSON Template" %}} -->
+
+Navigate to the **Config** tab of your robot's page in [the Viam app](https://app.viam.com).
+Click on the **Components** subtab and navigate to the **Create component** menu.
+Select **Raw JSON** mode and copy and paste the following:
```

Comment:
@andf-viam Ahhh, thank you for catching json here!! I think something really weird happens when commenting out tabs with hugo. It renders now for me in the GH preview, does it render for you?

Reply:
It does! And I think you're right about this being a very-hugo very-limitation. If not you, here, this would have been me, elsewhere!! Thanks for fixing!